packages/xod-arduino/src/transpiler.js

```diff
@@ -800,7 +820,7 @@ export const getPinsAffectedByErrorRaisers = def(
 );
 
 export const transformProject = def(
-  'transformProject :: Project -> PatchPath -> Liveness -> Either Error TProject',
+  'transformProject :: Project -> PatchPath -> Liveness -> XodVariables -> Either Error TProject',
```

Comment:
```suggestion
  'transformProject :: Project -> PatchPath -> TConfig -> Either Error TProject',
```

Keep it simple?

Reply:
`TConfig` is an internal type... See the answer above and let's discuss it there.